src/gis/components/Map.js

```diff
@@ -206,7 +214,9 @@ const Map = props => {
         enableSearch={props.enableSearch}
         enableDraw={props.enableDraw}
       />
-
+      {'undefined' !== typeof props.enableSearch && (
```

Comment:
Question: Do we need this? null and undefined are falsy so I think that should be enough for this

Reply:
I had inverted logic originally and "fixed" it this way. Thanks for flagging.